packages/components/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,136 @@
+     */
+    disabled?: boolean;
+    /**
+     * The aria label of the select component.
+     */
+    'aria-label': string;
+    /**
+     * The data test id of the select component.
+     */
+    'data-test-id'?: string;
+};
+
+export const SelectInput = (
+    {
+        children,
+        onSelect,
+        activeItem,
+        defaultValue,
+        placeholder = '',
+        disabled,
+        'aria-label': ariaLabel,
+        'data-test-id': dataTestId = 'fondue-select',
+    }: SelectComponentProps,
+    forwardedRef: ForwardedRef<HTMLDivElement>,
+) => {
+    const { inputSlots, menuSlots, items, defaultItem, clearButton } = useSelectData(children, defaultValue);
+
+    const { getToggleButtonProps, getMenuProps, getItemProps, reset, selectedItem, isOpen, highlightedIndex } =
+        useSelect({
+            items,
+            defaultSelectedItem: defaultItem,
+            selectedItem: activeItem,
+            onSelectedItemChange: ({ selectedItem }) => {
+                onSelect && onSelect(selectedItem);
+            },
+            itemToString: (item) => (item ? item.label : ''),
+        });
+
+    return (
+        <RadixPopover.Root open={isOpen}>
+            <RadixPopover.Anchor asChild>
+                <div
+                    className={styles.root}
+                    data-disabled={disabled}
+                    data-empty={!selectedItem}
+                    data-test-id={dataTestId}
+                    {...(disabled
+                        ? {}
+                        : getToggleButtonProps({
+                              'aria-label': ariaLabel,
+                              ...(forwardedRef ? { ref: forwardedRef } : {}),
+                          }))}
+                >
+                    <span className={styles.input}>{selectedItem ? selectedItem.label : placeholder}</span>
+                    {inputSlots}
+                    {clearButton && (
+                        <RadixSlot
+                            onClick={(event) => {
+                                event.stopPropagation();
+                                reset();
+                            }}
+                            className={styles.clear}
+                        >
+                            {clearButton}
+                        </RadixSlot>
+                    )}
+                    <div>
+                        <IconCaretDown size={16} className={styles.caret} />
+                    </div>
+                </div>
+            </RadixPopover.Anchor>
+
+            <SelectMenu highlightedIndex={highlightedIndex} getMenuProps={getMenuProps} getItemProps={getItemProps}>
+                {menuSlots}
+            </SelectMenu>
+        </RadixPopover.Root>
+    );
+};
+SelectInput.displayName = 'Select';
+
+export const ForwardedRefSelect = forwardRef<HTMLDivElement, SelectComponentProps>(SelectInput);
+
+// @ts-expect-error We support both Select and Select.Combobox as the Root
+export const Select: typeof SelectInput & {
```

Comment:
Deprecate the old component?
Can you add the migration guide also? (it was `Dropdown` so in the migration guide we need the explaination of the new `Select` vs new `Dropdown`)